modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Activities/components/Form/components/LeadListSection/LeadListSection.tsx

```diff
@@ -59,19 +59,17 @@ const LeadListSection = ({
 		}
 	};
 
-	return (
+	return selectedTypeActivity !== TypeActivityKey.EVENT ? (
```

Comment:
Hey @henriquelimadotcom. if the selectedTypeActivity is required to show the entired component, the best is to do this check outside this component.

Also, it's better to do using && instead ternary.